src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(Storage storage, FruitTransaction transaction) {
```

Comment:
According to the checklist, we should avoid beginning method implementations with an empty line. Remove the redundant empty line at the start of the `handle` method.